proxies/README.md

```diff
@@ -0,0 +1,329 @@
+
+<h1 id="devcycle-test-harness-proxy">DevCycle Test Harness - Proxy v1.0.0</h1>
+
+# Overview
+The SDK proxy server is responsible for wrapping the SDK under test with an HTTP interface that can be used to call any of its public-facing methods with an arbitrary set of arguments,
+and returning the response to the caller (which will be the harness server). It will also allow configuration options to be passed which override the base API URL the SDK makes requests to.
+Setting that URL to the test harness server’s address allows the harness to mock the backend APIs to return data back (or to trigger errors).
+
+### About `entityType`
+To check that the SDK methods are returning a correct type of an instance, they will all contain an `entityType` in their response. `enityType` can be one of four values:
+  `Variable`, `User`, `Feature`, `Object`
+
+<h1 id="devcycle-test-harness-proxy-client">Client</h1>
+
+SDK Client
+
+<a id="opIdcreateClient"></a>
+
+`POST /client`
+
+*Create a DVC client instance*
+
+> Body parameter
+
+```json
+{
+  "clientId": "string",
```

Comment:
any way to add descriptions for the individual request body fields?

Reply:
this is what I was referring to in the slack about it not being perfect. I spent some time this morning trying to get the descriptions for request bodies to show up in the markdown file but for some reason it doesn't. I'm thinking the spec might be wrong but when I use redocly to preview, everything looks good so I'm thinking it's something with widdershins. I'll dig into it a bit more but idk if it's worth spending more time on it